data/json/items/tool/toiletries.json

```
@@ -160,7 +160,7 @@
     "volume": "6 ml",
     "//": "This is calculated from medium weight cotton at 203 GSM, with a size of 6 inches square",
     "price": 200,
-    "price_postapoc": 50,
+    "price_postapoc": 5,
```

```suggestion
    "price_postapoc": 2,
```
Aren't these mass produced by butchering clothes?  If they take more work than that 5 cents is probably ok.

---

Yeah, they're mass-produced. I didn't want to overnerf but this seems even better on a second thought.